apis/nucleus/src/index.js

```diff
@@ -404,13 +405,29 @@ function nuked(configuration = {}) {
          * @typedef { boolean | 'toggle' } SearchMode
          */
 
+        /**
+         * @typedef { 'selectionConfirm' | 'selectionCancel' } SelectionEventTypes
+         */
+
+        /**
+         * Event listener function on instance
+         *
+         * @method
+         * @name FieldInstance#on
+         * @param {SelectionEventTypes} eventType event type that function needs to listen
```

Comment:
Don't call them SelectionEvents as it could be any type of event in the future. Better just use a generic FieldInstanceEventTypes